build.gradle

```diff
@@ -79,6 +84,8 @@ dependencies {
     modImplementation include("dev.onyxstudios.cardinal-components-api:cardinal-components-base:${project.cca_version}")
     modImplementation include("dev.onyxstudios.cardinal-components-api:cardinal-components-entity:${project.cca_version}")
 
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
+
```

Comment:
I will admit that the inclusion of annotation sources is not 100% consistent in this project. I would however much prefer to not include an extra dependency for the sole reason of bringing in annotations. The only missing annotation in this particular case is `@Nullable`, which can be sourced from the included JetBrains annotations dep (which in this project is used heavily already).